README.md

```diff
@@ -43,6 +43,46 @@ If you click *Plaintext* you should see something like this:
 5. Leave the default encryption key selected and click next.
 6. Give a name to your Secret and click next (for example: mySecret).
 
+
+## Private network setup
+The setup below is needed if the integration infrastructure is supposed to be deployed inside a VPC.
+
+### VPC setup
+Sample VPC setup cloudformation template: [vpc-setup.yml](sample-templates/vpc-setup.yml)
+
+
+### Snowflake Privatelink config and VPC id
+Get Snowflake PrivateLink configurations using the following command
+
+```
+select SYSTEM$GET_PRIVATELINK_CONFIG();
+```
+
+Response output will have following details:
```

Comment:
tiny nit: since you'll need to make changes below, can you please add a "the" here to say "...will have the following..."